end-to-end-test/local/specs/core/studyview.screenshot.spec.js

```diff
@@ -36,7 +36,7 @@ describe('study view generic assay categorical/binary features', function() {
         // and select a option
         $('div[data-test="GenericAssaySelection"]').waitForExist();
         $('div[data-test="GenericAssaySelection"] input').setValue(
-            'mutational_signature_category_10'
```

Comment:
I need to update this line since we use the first profile as default option, and the first profile is `pvalue profile`, and we cannot find `mutational_signature_category_10` anymore when we selected `pvalue profile`